test/E2ETests/project.json

```diff
@@ -21,6 +21,14 @@
                 "System.Net.Http": "",
                 "System.Xml": ""
             }
+        },
+        "dnxcore50": {
```

Comment:
running it on coreclr is a bit redundant on teamcity and its going to take more time.